server/system/types/user.go

```diff
@@ -30,6 +30,8 @@ type (
 		SuspendedAt *time.Time `json:"suspendedAt,omitempty"`
 		DeletedAt   *time.Time `json:"deletedAt,omitempty"`
 
+		AvatarInitialsMeta *UserAvatarInitialsMeta `json:"avatarInitialsMeta"`
```

Comment:
Are we storing this both in the PNG and the DB? If so, should we?

Reply:
We should store the fields `[initials, bgcolor, textcolor]` on the database to maintain form interactivity on the front end.